intelmq/bots/parsers/bambenek/parser.py

```diff
@@ -32,32 +33,33 @@ def parse_line(self, line, report):
             self.tempdata.append(line)
 
         else:
-            value = line.split(',')
+            m = re.match(r"(?P<ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}),(?P<description>.*), \
+                (?P<timestamp>\d{4}-\d{2}-\d{2}[ ]\d{2}[:]\d{2}),(?P<url>.*)", line)
+            values = m.groups()
```

Comment:
That line raises an exception in the tests

Reply:
It could be my line split. I'm not sure how to break the regex line into two lines in order not to trigger the "Line too long" warning when it comes to code style checking.

Reply:
Just split the line like this:
```python
            m = re.match(r"(?P<ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}),(?P<description>.*), "
                         r"(?P<timestamp>\d{4}-\d{2}-\d{2}[ ]\d{2}[:]\d{2}),(?P<url>.*)", line)
```